setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup
```

Comment:
Did you test this `setup.py`? I think it is only necessary if you want to package this repo (or is this intended?)

Reply:
This is also from the gym tutorial. I am not sure if its needed, so I kept it.